jwt/api.py

```diff
@@ -25,6 +25,25 @@ def __init__(self, algorithms=None):
             if key not in self._valid_algs:
                 del self._algorithms[key]
 
+        if not options:
+            options = {}
+
+        self.default_options = {
+            'verify_signature': True,
+            'verify_exp': True,
+            'verify_nbf': True,
+            'verify_iat': True,
+            'verify_aud': True,
+        }
+
+        try:
+            merged_options = dict()
+            for k, v in self.default_options.items():
+                merged_options[k] = options.get(k, v)
```

Comment:
The slight difference is that the with your way, `self.options` can end up with keys that weren't originally in `default_dict`.  It isn't a problem as long as we don't ever assume `merged_options` to only contain the default keys.